cmd/cloudx/project/create.go

```diff
@@ -51,6 +52,7 @@ func NewCreateProjectCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringP("name", "n", "", "The name of the project, required when quiet mode is used")
+	cmd.Flags().BoolP("use-project", "", false, "Uses the created project as default.")
```

Comment:
`BoolP` means flag + shorthand. Here the shorthand is empty, so use `Bool` instead.

`Use ... as default` sounds a bit strange to me, what about:

```
	cmd.Flags().BoolP("set-default", "", false, "Set the created project as the default.")
```

Reply:
I agree, that `Uses the created project as default.` sounds a bit strange.
I wanted to stay within the terminology established by the related command `use project`. How about, changing the help text like you suggested and keeping the flag as is?

Reply:
True, that makes sense from that standpoint. SGTM :)